neo4j.gemspec

```diff
@@ -27,12 +27,12 @@ A Neo4j OGM (Object-Graph-Mapper) for use in Ruby on Rails and Rack frameworks h
   s.rdoc_options = ['--quiet', '--title', 'Neo4j.rb', '--line-numbers', '--main', 'README.rdoc', '--inline-source']
 
   s.add_dependency('orm_adapter', '~> 0.5.0')
-  s.add_dependency('activemodel', '>= 4')
-  s.add_dependency('activesupport', '>= 4')
+  s.add_dependency('activemodel', '~> 5.0.0.beta3')
+  s.add_dependency('activesupport', '~> 5.0.0.beta3')
```

Comment:
We definitely want to continue to support `>= 4` for `activemodel`, `activesupport`, and `railties`.  Maybe when you're testing you can use `5.0.0.beta3` in your `Gemfile`?